src/main/webapp/app/forms/form-footer/form-footer.component.ts

```diff
@@ -10,7 +10,7 @@ import { ButtonSize } from 'app/shared/components/button.component';
 })
 export class FormFooterComponent {
     @Output() save = new EventEmitter<void>();
-    // eslint-disable-next-line @angular-eslint/no-output-native
+
     @Output() cancel = new EventEmitter<void>();
```

Comment:
The client style now complains here. Are you sure that we can remove the comment?

Reply:
Nah, I am trying to revert it right now, not sure which automatic save action removed it 😅

Reply:
eslint removed that line, thats probably also how the other one disappeared

Reply:
I have no changed the approach by directly changing the names, so the lintignore is not needed anymore and should not cause further issues in the future